.github/workflows/sycl-update-gpu-driver.yml

```diff
@@ -15,9 +21,25 @@ jobs:
     if: github.repository == 'intel/llvm'
     steps:
     - uses: actions/checkout@v4
+    - name: Determine parameters based on the day of the week
+      run: |
+        if [ "${{ github.event_name }}" = "workflow_dispatch" ]; then
+          # Use the input dayofweek when manually triggered
+          DAY_OF_WEEK="${{ github.event.inputs.dayofweek }}"
+        else
+          # Use the current day of the week when triggered by a schedule
+          DAY_OF_WEEK=$(date +%u)
+        fi
+
+        if [ "$DAY_OF_WEEK" = "2" ]; then
+          PARAM=""
+        else
+          PARAM="--igc-dev-only"
+        fi
+        echo "PARAM=$PARAM" >> $GITHUB_ENV
```

Comment:
I don't like this. I'd rather move common part into a re-usable workflow and then have two separate cron jobs in distinct .yml files. Also, why can't we just run the existing job frequently enough?

Reply:
Running existing job freq enough is also ok, but considering the situation of "red" CI -- it might not be realistic to push all driver update more than once per week. 
But sure, I am more than happy to do so if we want to try updating *ALL* drivers more than once a week.

Reply:
I *think* the reason we do it once a week is simply because released version aren't updated that frequently and we wanted to avoid running unnecessary task. That said, if nothing is there to update, it should be very fast, so I don't see any reasons for us not to do that.

\+ @bader  maybe?

Reply:
That's right. NEO driver was released one a week in the past. These days, NEO release cadence is once a month.